src/frontend/static/frontend/src/components/pipeline/all-experiments-view/ClinicalSourcePopup.tsx

```diff
@@ -378,12 +378,31 @@ export class ClinicalSourcePopup extends React.Component<PopupClinicalSourceProp
         this.props.onSuccessCallback()
     }
 
+    /**
+     * Validate survivals columns are corrected formatted.
+     * @returns boolean validation return true if correct
+     */
+
+    validateSurvivalsColumnsComplete () {
```

Comment:
1. Agregar tipo de retorno a la función! Devuelve un `boolean`. 
2. Sacar esa línea en blanco que está en la línea 385, la documentación debe ir pegada a la función para que los IDEs reconozcan la documentación y la muestren formateada.
3. Por útlimo y más importante! Toda esta función se puede reducir a un simple `return this.state.survivalColumns.some((item) => ...)` que tiene early stopping y encima queda más manenible

Reply:
Dale gena, ahi acomode el refactor de esto!